test/bin/scenario.sh

```diff
@@ -259,6 +269,8 @@ SSH_PRIV_KEY: "${SSH_PRIVATE_KEY:-}"
 SSH_PORT: ${ssh_port}
 EOF
 
+    trap "sos_report" EXIT
```

Comment:
This should move to `action_run` for consistency with the above comment.

Reply:
Moved it to the top, as it may be useful from the beginning.